client/src/AppBundle/Controller/Admin/IndexController.php

```diff
@@ -231,14 +237,13 @@ public function deleteConfirmAction($id)
         /** @var EntityDir\User $loggedInUser */
         $loggedInUser = $this->getUser();
 
-        if (!$loggedInUser->isAdminOrSuperAdmin()) {
-            $message = $userToDelete->isAdminOrSuperAdmin() ? 'Only Super Admins can delete Admins or Super Admins' : 'Only Admins or Super Admins can delete users';
-            throw new DisplayableException($message);
-        }
+        $token = new UsernamePasswordToken($loggedInUser, 'credentials', 'memory');
+        $canDelete = $this->userVoter->vote($token, $userToDelete, [UserVoter::DELETE_USER]);
```

Comment:
Another helper: this one will automatically generate a 403 if the grant isn't met.
```suggestion
        $this->denyAccessUnlessGranted('delete-user', $userToDelete, 'Unable to delete this user');
```

Reply:
Cool, makes sense here 👍